composer.json

```diff
@@ -22,7 +22,7 @@
     "require": {
         "php": ">=7.2",
         "ext-json": "*",
-        "guzzlehttp/guzzle": "~6.0"
+        "guzzlehttp/guzzle": "^7.0"
     },
     "require-dev": {
         "ext-xdebug": "*",
```

Comment:
I've got a issue with this php module on my CI server : https://scrutinizer-ci.com/g/defro/google-streetview/inspections/88d1a0ce-596e-4cfd-8359-eae5bc266d8b
Can you just remove this line please ? and I will know if unit test will succeed. Thanks